universum/main.py

```diff
@@ -72,6 +72,10 @@ def execute(self) -> None:
 
         self.vcs.prepare_repository()
         project_configs: Configuration = self.launcher.process_project_configs()
+        if not project_configs:
```

Comment:
The issues with this approach are the following:
1. Nonci mode is not considered. It could be intentional or not, but the intention is not expressed in the source code.
2. Logically configs should be handled by launcher, not main
3. Because of item 2, the SilentAbortException is used instead of CriticalCiException. The latter is better choice in general case.